fivegmag_ExoDvbi_player/replace.py

```diff
@@ -1,30 +0,0 @@
-import glob, os, shutil
```

Comment:
You have deleted this file, however it is still referenced in the README.md (line 48). 

How is the patch file now being applied?

Reply:
I see your next commit addressed the inclusion, but that still leaves the question of how this patch is applied as I see no instruction for that.

Reply:
Hello, thank you for spotting this. I've modified the README file as requested, please let me know if anything else.